### PR TITLE
Titles updated

### PR DESCRIPTION
**Pull request by Gasta88**
I've changed the title of all the .md files that were 9 months old.

### CODE REVIEW

File: wiki/ACE_contig_to_alignment.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: ACE contig to alignment
+title: Visualize an alignment from conting archived in ACE files.
```
Comment (peterjc) [inline wiki/ACE_contig_to_alignment.md]: Typo: contig

I think visualise isn't quite the right description here, maybe leave it as it was?

File: wiki/MotifDev.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: MotifDev
+title: Current development for the MotifDev module.
```
Comment (peterjc) [inline wiki/MotifDev.md]: The module is just Bio.Motif rather than MotifDev (short for Motif development)

File: wiki/ProtParam.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-title: ProtParam - Analyzing Protein Sequences
+title:Analyzing protein sequences with the ProtParam module.
```
Comment (peterjc) [inline wiki/ProtParam.md]: Should there be a space after ``title:``?

**Comment by peterjc** [comment]
I've made some minor comments, otherwise this looks good - thank you.

**Comment by Gasta88** [comment]
Thanks for taking the time and commenting them.

Shall I update more recent titles?

**Comment by peterjc** [comment]
If you see any more which you think need changing, that's be great.

**Pull request merged**